### PR TITLE
Sumeragi Messages in the message module

### PR DESCRIPTION
**Pull request by eadventurous**
Signed-off-by: Egor Ivkov <e.o.ivkov@gmail.com>

### CODE REVIEW

File: iroha/src/lib.rs [hunk 1]
```diff
@@ -30,7 +30,7 @@ use crate::{
     peer::Peer,
     prelude::*,
     queue::Queue,
-    sumeragi::{Message, Sumeragi},
+    sumeragi::{message::Message as SumeragiMessage, Sumeragi},
```
Comment (humb1t) [inline iroha/src/lib.rs:33]: It's still the only message here) Please do not alias it.

File: iroha/src/sumeragi.rs [hunk 2]
```diff
@@ -708,6 +540,197 @@ impl Debug for Sumeragi {
...
+    ValidatingPeer,
+    /// Observing Peer.
+    ObservingPeer,
+    /// Proxy Tail.
+    ProxyTail,
+}
+
+/// Contains message structures for p2p communication during consensus.
+pub mod message {
+    use super::*;
```
Comment (humb1t) [inline iroha/src/sumeragi.rs:558]: Maybe I missed something looking through - but do we need something from sumeragi here? 
If not - it's better to place normal imports here. That way we will see how much entities are connected in module and submodule.

File: iroha/src/torii.rs [hunk 3]
```diff
@@ -1,7 +1,7 @@
 //! This module contains incoming requests handling logic of Iroha.
 //! `Torii` is used to receive, accept and route incoming instructions, queries and messages.
 
-use crate::{prelude::*, sumeragi::Message, MessageSender};
+use crate::{prelude::*, sumeragi::message::Message as SumeragiMessage, MessageSender};
```
Comment (humb1t) [inline iroha/src/torii.rs:4]: Please, do not use alias for sumeragi message.

File: iroha/src/lib.rs (hunk 1 above)
Comment (eadventurous) [reply iroha/src/lib.rs:33]: Sure, though still, it seem more intuitive for me with `SumeragiMessage` :sweat_smile:

**Pull request merged**